keps/prod-readiness/sig-scheduling/5254.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 5254
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
I'm not sure whether @johnbelamaric has time - I doubt it.

Let's see who from the PRR reviewers takes this, if it's not too late.